contrib/educational_decoder/README.md

```diff
@@ -0,0 +1,18 @@
+Educational Decoder
+===================
+
+`zstd_decompress.c` is a self-contained implementation of a decoder according
+to the Zstandard format specification written in C99.
```

Comment:
The reference implementation is written in C,
but the format specification is just plain markdown.
Better link to the specification, as you do below.